src/lib/peer.js

```diff
@@ -1,11 +1,13 @@
 var ee2 = require('eventemitter2').EventEmitter2;
 var io = require('socket.io-client');
-var bows = require('bows'); 
 var Id = require('dht-id');
 var FingerTable = require('./finger-table.js');
 var ChannelManager = require('./channel-manager.js');
 
-log = bows('webrtc-explorer');
+if (typeof localStorage === "undefined" || localStorage === null) {
+    var LocalStorage = require('node-localstorage').LocalStorage;
+    localStorage = new LocalStorage('./scratch');
+}
```

Comment:
That's one of the node-js related changes as mentioned in the initial post. Still, there should be no compatibility issues with running in the browser